brax/v2/io/mjcf.py

```diff
@@ -393,6 +393,7 @@ def load_model(mj: mujoco.MjModel) -> System:
         'transform': Transform(pos=mj.geom_pos[i], rot=mj.geom_quat[i]),
         'friction': mj.geom_friction[i, 0],
         'elasticity': custom['elasticity'][i],
+        'rgba': mj.geom_rgba[i], # if there is no colour specified, default is applied
```

Comment:
Thanks for doing this!
Can we make the default rgba="0.4 0.33 0.26 1", to match the current brax viz, only if a user default is not specified? You might have to read this from the xml itself to see if a default was specified by the user.
In other words: If a default is specified by the user, let's use that new default, otherwise let's use the brax default instead of the mjcf default of "0.5, 0.5, 0.5, 1."

Reply:
You are welcome! I can check if I could make that happen. Soon I will apply and test all the corrections and apply the  default if proceeds!

Reply:
Hello!

There is some part in load_model() that would be (not final implementation, some  guidance of the results in the prints):
```
    # (default_rgba info got from XML)
    if default_rgba:
      print(f'default provided, using {mj.geom_rgba[i]}')
    else:
      # mujoco default colour will be applied by mujoco reader
      """
      edge case if [0.5 0.5 0.5 1.] is provided in a children geom it would get overwritten
      there is no way of knowing if the value of mj.geom_rgba[i] is provided by user or mujoco
      unless cheching XML here for each of the geoms
      """
      if (mj.geom_rgba[i] == [0.5, 0.5,  0.5, 1. ]).all():
        print(f'default provided, using {[0.4, 0.33, 0.26, 1]} instead {mj.geom_rgba[i]}')
```
Checking the edge case may be too much and I think checking XML and the MuJoCo at the same time may be some technical debt. Should I skip the edge case and just proceed? I would move on the texture integration once this PR gets close.

Reply:
maybe just this would do for now without getting `default_rgba` from the xml:

`'rgba': [0.4, 0.33, 0.26, 1.0] if (mj.geom_rgba[i] == [0.5, 0.5,  0.5, 1. ]).all() else mj.geom_rgba[i]`

Reply:
That will upset the users that really want [0.5, 0.5,  0.5, 1. ] but we can live with that! :laughing:

Reply:
@ManuCorrea could you push your latest commits? seems like most resolved comments are not resolved in the code, but LGTM otherwise

Reply:
Yes! Sorry I didn't want to push the partially resolved stuff until having all and tested, I have been busy recently.